MobileWallet/Screens/Send/AddRecipient/ContactsTableViewController.swift

```diff
@@ -54,8 +54,17 @@ class ContactsTableViewController: UITableViewController {
                 filteredRecentContactList =  recentContactList
                 filteredContactList = contactList
             } else {
-                filteredRecentContactList = recentContactList.filter { $0.alias.0.localizedCaseInsensitiveContains(filter) }
-                filteredContactList = contactList.filter { $0.alias.0.localizedCaseInsensitiveContains(filter) }
+                if filter.containsEmoji {
+                    filteredRecentContactList = recentContactList.filter {
+                        ($0.publicKey.0?.emojis.0.localizedCaseInsensitiveContains(filter.emojiString) ?? false)
+                    }
+                    filteredContactList = contactList.filter {
+                        ($0.publicKey.0?.emojis.0.localizedCaseInsensitiveContains(filter.emojiString) ?? false)
+                    }
+                } else {
+                    filteredRecentContactList = recentContactList.filter { $0.alias.0.localizedCaseInsensitiveContains(filter) }
+                    filteredContactList = contactList.filter { $0.alias.0.localizedCaseInsensitiveContains(filter) }
+                }
```

Comment:
If you combine the `else` and `if` here I think it will solve the search issue. Something like:

```swift
filteredRecentContactList = recentContactList.filter{ 
    $0.publicKey.0?.emojis.0.localizedCaseInsensitiveContains(filter.emojiString)
    || $0.alias.0.localizedCaseInsensitiveContains(filter)
}
```

Reply:
Or you can also go for the one below for a more straightforward search:
```swift
filteredRecentContactList = recentContactList.filter { 
    $0.publicKey.0?.emojis.0.localizedCaseInsensitiveContains(filter)
    || $0.alias.0.localizedCaseInsensitiveContains(filter)
}
```